python/py-bidi/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-bidi
+python.rootname     python-${python.rootname}
```

Comment:
generating the `python.rootname` variable with the "previous/current" `python.rootname` variable is confusing, please just spell this out

Reply:
Thanks done.